src/index.js

```diff
@@ -32,17 +30,14 @@ const theme = createMuiTheme({
 })
 
 const root_element = 'root'
-const jss = create(jssPreset())
 
 ReactDOM.render(
   <I18nextProvider i18n={i18n}>
-    <JssProvider jss={jss}>
-      <MuiThemeProvider theme={theme}>
+      <ThemeProvider theme={theme}>
```

Comment:
Everything looks right in here theme-wise. Unfortunately, the calendar is now back to its default color. This makes me very sad :( 

I guess I'll have to go update that project sometime...